pkg/metricscollector/v1beta1/file-metricscollector/file-metricscollector.go

```diff
@@ -52,7 +65,7 @@ func CollectObservationLog(fileName string, metrics []string, filters []string,
 	case commonv1beta1.JsonFormat:
 		return parseLogsInJsonFormat(strings.Split(logs, "\n"), metrics)
 	}
```

Comment:
```suggestion
	default:
		return nil, nil
	}
```

Reply:
I see!

Reply:
Sorry, it seems okay for this change to make the switch clause tidy and neat. Plz ignore what I say in the comments below🤣.

Reply:
No worries.